crane-cloud/frontend
# credits user side

# Description
Adds credits designs to the frontend along side flutter wave
## Type of change
Please delete options that are not relevant.

## Trello Ticket ID

https://trello.com/c/GEAq4Z3w

## How Can This Been Tested?

pull the branch and notice the changes and newly added credit designs, and also on the billing page, try to pay bills  and select credits. if you select flutterwave and the cant proceed then the cluster dint bring back the total values.
Note: the payment model design was changed abit due to  initial implementation of flutterwave 


## Checklist:


# Screenshots

design was changed from. 
![image](https://user-images.githubusercontent.com/69305400/179727257-2371b200-dfc2-4966-bd47-59a55e6ec850.png)
to due to flutterwave implementation
![image](https://user-images.githubusercontent.com/69305400/179727501-fba11575-d660-4569-a932-eb11bc61e538.png)
